Controller/ApiController.php

```diff
@@ -50,16 +51,26 @@ public function __construct(
     }
 
     /**
-     * @Route("/api", name="api", methods={"POST"})
+     * @Route("/api", name="api", methods={"GET", "POST"})
      * @Security("has_role('ROLE_OAUTH2_READ')")
      */
     public function index(Request $request)
     {
-        $body = json_decode($request->getContent(), true);
-        $schema = $this->schema;
-        $query = $body['query'];
-        $variableValues = isset($body['variables']) ? $body['variables'] : null;
-        $result = GraphQL::executeQuery($schema, $query, null, null, $variableValues);
+        switch ($request->getMethod()) {
+            case 'GET':
+                $query = $request->get('query');
+                $variableValues = $request->get('variables');
```

Comment:
`variables` はjsonで来るので、arrayに変換してやる必要があると思います。

```suggestion
                $variableValues = json_decode($request->get('variables'), true);
```

テストクエリ

```
curl -X GET -H "Authorization: Bearer ${ACCESS_TOKEN}" http://127.0.0.1:8000/api\?query\=mutation+UpdateShipped%28%24id%3AID%21%29%7BupdateShipped%28id%3A%24id%29+%7Bid+shipping_date%7D%7D\&variables\=%7B%22id%22%3A+1554%7D
```

結果

```
{"data":{"updateShipped":{"id":"1554","shipping_date":"2020-07-30T06:14:03+00:00"}}}
```